docs/03-tutorials/00-application-connectivity/ac-02-get-client-certificate.md

```diff
@@ -149,20 +189,18 @@ A successful call returns the following response:
 }
 ```
 
-Use `urls.metadataUrl` and `urls.eventsUrl` to get the URLs to the Application Registry API and to the Event Publisher API.
+Use **urls.metadataUrl** and **urls.eventsUrl** to get the URLs to the Application Registry API and to the Event Publisher API.
 
-## Call Application Registry and Event Publisher on local deployment
-
-Since the local Kyma installation uses the self-signed certificate by default, skip TLS verification.
-
-Call Application Registry with this command:
-
-```bash
-curl https://gateway.local.kyma.dev/{APP_NAME}/v1/metadata/services --cert {CLIENT_CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -k
-```
```

Comment:
I removed this part, 'cause we don't want to have commands for local deployment specifically, just mention to skip cert verification (we have such a note at the top of the document), and if we turn it into a general command working for both a local and remote cluster, it becomes the same as the command in the previous section.